addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -15,7 +15,7 @@ publicWidget.registry.crmPartnerAssign = publicWidget.Widget.extend({
         'click .new_opp_confirm': '_onNewOppConfirm',
         'click .edit_opp_confirm': '_onEditOppConfirm',
         'change .edit_opp_form .next_activity': '_onChangeNextActivity',
-        'click div.input-group span.fa-calendar': '_onCalendarIconClick',
```

Comment:
Actually for stable, you should keep the old JS. If someone keeps the old layout but refreshes it browse, he may run into issues. What you can do is add a comment that this code is to remove in master postv15 .